pkg/render/intrusiondetection/dpi/dpi.go

```diff
@@ -180,6 +183,25 @@ func (d *dpiComponent) dpiDaemonset() *appsv1.DaemonSet {
 	if d.cfg.TyphaNodeTLS.NodeSecret.UseCertificateManagement() {
 		initContainers = append(initContainers, d.cfg.TyphaNodeTLS.NodeSecret.InitContainer(DeepPacketInspectionNamespace))
 	}
+	if d.cfg.IntrusionDetection.Spec.DeepPacketInspectionDaemonset.SnortInitContainers != nil {
```

Comment:
I think this will cause a panic if `d.cfg.IntrusionDetection.Spec.DeepPacketInspectionDaemonset` is `nil`

Reply:
Corrected as well.